libobs/data/format_conversion.effect

```diff
@@ -535,6 +559,19 @@ float4 PSPlanar444A_Reverse(FragPos frag_in) : TARGET
 	return rgba;
 }
 
+float4 PSPlanar444A_12LE_Reverse(FragPos frag_in) : TARGET
+{
+    int3 xy0 = int3(frag_in.pos.xy, 0);
+    float y = image.Load(xy0).x;
+    float cb = image1.Load(xy0).x;
+    float cr = image2.Load(xy0).x;
+    float alpha = image3.Load(xy0).x * 16.;
+    float3 yuv = float3(y, cb, cr);
+    yuv *= 16.;
+    float4 rgba = float4(YUV_to_RGB(yuv), alpha);
+    return rgba;
```

Comment:
`rgba.rgb = srgb_nonlinear_to_linear(rgba.rgb);` for the switch to GS_RGBA16F, which is a linear target.